keras_nlp/__init__.py

```diff
@@ -27,4 +27,4 @@
 from keras_nlp import tokenizers
 from keras_nlp import utils
 
-__version__ = "0.5.0"
+__version__ = "0.5"
```

Comment:
We should always keep the patch release number.

The branch is `r0.5` as it will be continued to update patch release to patch release. (See core keras for example).

But the version numbers we expose from the library should always follow https://semver.org/, with three parts.

Reply:
Also, if we are cutting more dev releases from here, should we keep a devX postfix for now?

Reply:
sg!